x/emissions/keeper/block_rewards_components_test.go

```diff
@@ -43,8 +54,135 @@ func TestKeeper_CalculateFixedValidatorRewards(t *testing.T) {
 	for _, tc := range tt {
 		t.Run(tc.name, func(t *testing.T) {
 			blockRewards, err := emissionskeeper.CalculateFixedValidatorRewards(tc.blockTimeInSecs)
+			if tc.wantErr {
+				require.Error(t, err)
+				return
+			}
 			require.NoError(t, err)
 			require.Equal(t, tc.expectedBlockRewards, blockRewards)
 		})
 	}
 }
+
+func TestKeeper_GetFixedBlockRewards(t *testing.T) {
+	k, _, _, _ := keepertest.EmissionsKeeper(t)
+	fixedBlockRewards, err := k.GetFixedBlockRewards()
+	require.NoError(t, err)
+	expected, err := emissionskeeper.CalculateFixedValidatorRewards(emissionstypes.AvgBlockTime)
+	require.NoError(t, err)
+	require.Equal(t, expected, fixedBlockRewards)
+}
```

Comment:
It might be a good idea to use a value for assertion , instead of using the `CalculateFixedValidatorRewards` function .